eoscostarica/guide.eoscostarica.io
# feat(frontend): GitHub Guide 257

### GH Issue

Resolves #257 

### Steps to test
1. Go to GitHub Guide
2. Check content is right.


#### CheckList
- [ ] The content is equal to https://guias.eoscostarica.io/docs/tutoriales/guia-github
